packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4032,8 +4032,12 @@ export default class Meeting extends StatelessWebexPlugin {
           `Meeting:index#usePhoneAudio --> removing audio track from desktop`
         );
 
-        // removing audio from desktop to avoid echo as phone audio is being used
-        Media.stopTracks(this.mediaProperties.audioTrack);
+        // removing local audio from desktop to avoid echo as phone audio is being used
+        return Media.stopTracks(this.mediaProperties.audioTrack);
+      })
+      .then(() => {
+        // removing remote audio from desktop to avoid echo as phone audio is being used
+        return Media.stopTracks(this.mediaProperties.remoteAudioTrack);
```

Comment:
this is just going to stop the track but the connection is still active you will need to do updateAudio with null and receiveAudio to false

Reply:
I don't think we should be stopping any audio (remote or local) when usePhoneAudio() is called. The call to updateAudio() should be done by the client app - that's how it works in the web app. SDK should remain flexible and allow the client to have both computer audio and phone audio if they want to.

Looks like I've missed the previous PR https://github.com/webex/webex-js-sdk/pull/2759/ - I would have made the same comment there and not approved it

Reply:
Got it. Will revert the changes of my last PR and will update the documentation as well. Thanks @marcin-bazyl @arun3528

Reply:
Wiki created: https://github.com/webex/webex-js-sdk/wiki/Use-phone-audio-for-SDK-meeting---Dial-IN-OUT

Reply:
the PR description needs to be updated to say that it's a revert and what is being reverted, right now the PR description is totally opposite to what the code changes are actually doing